archinstall/lib/user_interaction.py

```diff
@@ -334,12 +380,18 @@ def select_language(options, show_only_country_codes=True):
 		for index, language in enumerate(languages):
 			print(f"{index}: {language}")
 
-		print(' -- You can enter ? or help to search for more languages, or skip to use US layout --')
-		selected_language = input('Select one of the above keyboard languages (by number or full name): ')
+		# Current workaround for passing `generic_select`,
+		# if these values are provided as input
+		languages.extend(['?', 'help'])
+		languages_length = len(languages)
+
+		print(f' -- You can enter ? ({languages_length - 2}) or help ({languages_length - 1}) to search for more languages, or skip to use US layout --')
+		selected_language = generic_select(languages, 'Select one of the above keyboard languages (by number or full name): ',
+										  options_output=False)
 		
-		if len(selected_language.strip()) == 0:
+		if not selected_language:
 			return DEFAULT_KEYBOARD_LANGUAGE
-		elif selected_language.lower() in ('?', 'help'):
```

Comment:
By getting rid of lower here, Help and HELP are no longer considered valid. I'm not sure if I like this change

Reply:
Because of the workaround on line 385, the user can only enter '?' and "help", as I thought that the user would enter only such values or simply select these items in numbers